src/objects/Coin.ts

```diff
@@ -21,10 +21,10 @@ const entityConfig: EntityConfigType = {
   collisionCategory: CollisionCategories.coin,
   animations: [
     {
-      animationKey: 'spin',
+      animationKey: 'idle',
       fps: 12,
       start: 0,
-      end: 5,
+      end: 4,
```

Comment:
fix warning

Reply:
fucking dam it array starts at 0 lol 😖 Nice catch